### ISSUE TITLE
Prevent committing when there are no changes

### ISSUE THREAD

**Issue by jikkujose** [cloudaper/sketch-git-hooks#4]
Problem:

- Commit a change
- Immediately issue another commit (possible because the sketch file will be shown as not added to the repo)
- A new commit will appear without any file change in it

**Comment by stephenedmondson** [comment]
+1 Seeing the same thing.

**Comment by krystof-k** [comment]
Yes, that's probably happening. Should be possible to solve, I'll try to look into this.
